rita/Cargo.toml

```diff
@@ -63,3 +63,4 @@ handlebars = "1.0.3"
 byteorder = { version = "1.2.6", features = ["i128"] }
 openssl-probe = "0.1.2"
 num-traits="0.2"
+netstat="0.7.0"
```

Comment:
looks like this requires some sort of native dependency to link against. I'll have to do more research about exactly what.

Reply:
is this still needed now?